draft-ietf-quic-transport.md

```diff
@@ -4450,7 +4498,8 @@ with the Stream ID set appropriately. A receiver could use the current offset of
 data consumed to determine the flow control offset to be advertised.  A receiver
 MAY send MAX_STREAM_DATA frames in multiple packets in order to make sure that
 the sender receives an update before running out of flow control credit, even if
-one of the packets is lost.
+one of the packets is lost. As soons as the packet with the MAX_STREAM_DATA
+frame is acknowlegded, there is no need to send the same max stream limit agian.
```

Comment:
Was this not obvious before?

Reply:
the point here is that I am actually unhappy with the "in multiple packets". How much is multiple? i would assume something like 3 but others might think they are only on the safe side if they send it e.g. 20 times... I just wanted to make sure nobody gets the idea that it is most safe to just always send it. I would be more happier if you could just give a number instead of saying "in multiple packets".

Reply:
I think sending the current value in each packet is a reasonable strategy, albeit verbose if you have a lot of streams.  It's certainly a good, even recommended, optimization to suppress that if it hasn't changed and one of them has been ACK'd.  But I think the larger point is that it's okay to duplicate, and how much to duplicate is entirely under the implementation's control.

Also, "again."

Reply:
"soons" "acknowlegded"

The new text is redundant though.